django/db/models/indexes.py

```diff
@@ -14,6 +14,9 @@ class Index(object):
     suffix = 'idx'
 
     def __init__(self, fields=[], name=None):
+        # Sanity checks for fields argument
+        if type(fields) != list:
```

Comment:
`isinstance(fields, (list, tuple))` seems more appropriate.

I don't think the comment is needed.